### PR TITLE
Setup - Add maker-pending-shifts maker-history-shifts (components, .ts files)

### PR DESCRIPTION
**Pull request by frankdamico**
- revised maker-unfulfilled-shifts

### CODE REVIEW

File: src/app/maker/maker-create-shift/maker-create-shift.component.html [hunk 1]
```diff
@@ -28,7 +28,8 @@ <h3>Create a Shift</h3>
     <p></p>
     <ion-textarea placeholder="Enter description: Formal party at private residence. White dress shirt required."></ion-textarea> 
       <textarea rows="5" placeholder="Enter event description  "></textarea>
-  </ion-item>
+
+  </ion-item shift-positions>
     <p></p>
     <select name="Positions">
```
Comment (alexa-w) [inline src/app/maker/maker-create-shift/maker-create-shift.component.html:33]: change to `ion-select`

File: src/app/maker/maker-history-shifts/maker-history-shifts.component.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<p>maker-history-shifts works!</p>
```
Comment (alexa-w) [inline src/app/maker/maker-history-shifts/maker-history-shifts.component.html]: remove this

File: src/app/maker/maker-history-shifts/maker-history-shifts.component.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+<ion-content makerShifts-history>
+  <ion-title><h3>Shifts History</h3></ion-title>
+  <ul>
```
Comment (alexa-w) [inline src/app/maker/maker-history-shifts/maker-history-shifts.component.html]: change to `ion-list`

File: src/app/maker/maker-history-shifts/maker-history-shifts.component.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    <!-- v-- hrefs below - CHANGE to ionic or ang routerLink --v -->
+    <li><ion-card><a href><h4>Anniversary Brunch</h4><p></p>
```
Comment (alexa-w) [inline src/app/maker/maker-history-shifts/maker-history-shifts.component.html]: these hrefs should become function calls eventually, but they're fine dummy entries at the moment

File: src/app/maker/maker-history-shifts/maker-history-shifts.component.html [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+      6/6 filled | 02/21/10:0A - 12:00P</a>
+    </ion-card></li>
+      <p></p>
+    <li><ion-card><a href><h4>Mayoral Fundraising Party</h4><p></p>
+      5/5 filled | 02/29 10:00P - 2:00A</a>
+    </ion-card></li>
+  </ul>
+  <!-- This href goes to RATE WERKERS -->
+  <button><ion-button href="/maker-home">RATE WERKERS</ion-button></button>
```
Comment (alexa-w) [inline src/app/maker/maker-history-shifts/maker-history-shifts.component.html]: Remove `<button>` tags
href should eventually become click event

File: src/app/maker/maker-pending-shifts/maker-pending-shifts.component.html [hunk 6]
```diff
@@ -0,0 +1,14 @@
+<p>maker-pending-shifts works!</p>
+<ion-content makerShifts-pending>
+  <ion-title><h3>Shifts Pending</h3></ion-title>
+    <ul>
```
Comment (alexa-w) [inline src/app/maker/maker-pending-shifts/maker-pending-shifts.component.html:4]: Change to `ion-list`

File: src/app/maker/maker-search/maker-search.component.html [hunk 7]
```diff
@@ -44,5 +42,12 @@ <h3>Invite Werkers</h3>
       <li><a href>Fav3 Werker3</a>  <button ion-button class-name="button-accept">INVITE</button></li>
     </ul>
   </div>
+  <div class-name=werkerSearch>
+      <ul>
```
Comment (alexa-w) [inline src/app/maker/maker-search/maker-search.component.html:46]: change to `ion-list`

File: src/app/maker/maker-search/maker-search.component.html [hunk 8]
```diff
@@ -44,5 +42,12 @@ <h3>Invite Werkers</h3>
...
+        <li><a href>Werker4</a>  <button ion-button class-name="button-accept">INVITE</button></li>
```
Comment (alexa-w) [inline src/app/maker/maker-search/maker-search.component.html:47]: change `button ion-button`s to just `ion-button`

File: src/app/maker/maker-unfilled-shifts/maker-unfilled-shifts.component.html [hunk 9]
```diff
@@ -1,12 +1,14 @@
...
-      <li><a href><h4>Easter Brunch</h4><p></p>
-        4/6 filled | 04/21/10:0A - 12:00P</a></li>
-        <p></p>
-        <li><a href>Bachelourette Party<p></p>
-          2/5 filled | 04/29 10:00P - 2:00A</a></li>
-    </ul>
-  </div>
+<ion-content makerShifts-unfulfilled>
+  <ion-title><h3>Shifts Unfulfilled</h3></ion-title>
+  <ul>
```
Comment (alexa-w) [inline src/app/maker/maker-unfilled-shifts/maker-unfilled-shifts.component.html:4]: change to `ion-list`

File: src/app/user/pendingshifts/pendingshifts.component.html [hunk 10]
```diff
@@ -1,3 +1,18 @@
-<p>
-  pendingshifts works!
-</p>
+<p>user-pendingshifts works!</p>
```
Comment (alexa-w) [inline src/app/user/pendingshifts/pendingshifts.component.html]: remove this

File: src/app/user/pendingshifts/pendingshifts.component.html [hunk 11]
```diff
@@ -1,3 +1,18 @@
...
+<div class-name=userShifts-pending>
+    <ion-title>
+      <h3>Shifts Pending</h3>
```
Comment (alexa-w) [inline src/app/user/pendingshifts/pendingshifts.component.html]: :+1: on these titles

File: src/app/user/pendingshifts/pendingshifts.component.html [hunk 12]
```diff
@@ -1,3 +1,18 @@
...
+    </ion-title>
+    
+    <ul>
```
Comment (alexa-w) [inline src/app/user/pendingshifts/pendingshifts.component.html]: change to `ion-list`

File: src/app/user/shift-details/shift-details.component.html [hunk 13]
```diff
@@ -1,3 +1,33 @@
-<p>
-  shift-details works!
-</p>
+<p>shift-details works!</p>
```
Comment (alexa-w) [inline src/app/user/shift-details/shift-details.component.html]: remove this

File: src/app/user/shift-details/shift-details.component.html [hunk 14]
```diff
@@ -1,3 +1,33 @@
...
+  <h3>Shift Details</h3>
+  <div class-name="shift-expanded">
+    <div class-name="shift-name">Shift Name</div>
+    <div class-name="shift-address">123 Street, City, ST 70111</div>
+    <div class-name="shift-time_date">08/21/19 7:00P - 12:30A</div>
+    <div class-name="shift-description"><p>Description: Catering private party for company anniversary. - Black dress clothes.</p></div>
+
+    <div class-name="filled-positions">
+    <h3>Shift Werkers</h3>
+      <ul>
```
Comment (alexa-w) [inline src/app/user/shift-details/shift-details.component.html]: change to `ion-list`

File: src/app/user/shift-details/shift-details.component.html [hunk 15]
```diff
@@ -1,3 +1,33 @@
-<p>
-  shift-details works!
-</p>
+<p>shift-details works!</p>
+<div>
+  <h3>Shift Details</h3>
+  <div class-name="shift-expanded">
+    <div class-name="shift-name">Shift Name</div>
+    <div class-name="shift-address">123 Street, City, ST 70111</div>
+    <div class-name="shift-time_date">08/21/19 7:00P - 12:30A</div>
```
Comment (alexa-w) [inline src/app/user/shift-details/shift-details.component.html:6]: nice formatting on that time!

**Review by alexa-w — CHANGES_REQUESTED** [review]

File: src/app/maker/maker-create-shift/maker-create-shift.component.html (hunk 1 above)
Comment (frankdamico) [reply src/app/maker/maker-create-shift/maker-create-shift.component.html:33]: 26-31 ion-select

**Pull request merged**